### ISSUE TITLE
Kick bug

### ISSUE THREAD

**Issue by guhmaster52** [CharonXXIII/Pixel-Worlds-C-Cheat#6]
Whenever you join a world with players, it kicks you for some reason.

**Comment by CharonXXIII** [comment]
Yeah, I won't be fixing it, I'll be having fun without any cheaters in Pixel Worlds with good cheats except less than 5 people.
